src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { ITodo } from '../../types/types';
+import Todo from '../Todo/Todo';
+
+interface TodoListProps {
+  userId: number;
+  todos: ITodo[];
+}
+
+const TodoList: React.FC<TodoListProps> = ({ userId, todos }) => {
```

Comment:
```suggestion
export function TodoList({userId, todos}: TodoListProps) {
```
Это субъективно и на вкус, но я компоненты через function препочитаю писать, там короче типизировать.